setup.py

```diff
@@ -169,6 +169,6 @@ def install_triton_client():
     DEPENDENCY_LINKS=[TORCH_INSTALL_URL, TORCHVISION_INSTALL_URL],
     entry_points='''
         [console_scripts]
-        modelci=modelci.cli:cli
+        modelci=modelci.cli:typer_click_object
```

Comment:
has this been well verified?

Reply:
yes,  the entrypoint for ModelCI CLI interface has been changed to `typer_click_object` function, `cli` function is no longer avaliable, https://github.com/cap-ntu/ML-Model-CI/blob/8ba8605b3bf8547ec37f6ad002c7706c7620e1b2/modelci/cli/__init__.py#L41-L42